src/foam/nanos/http/AuthWebAgent.java

```diff
@@ -229,6 +229,7 @@ public Session authenticate(X x) {
             session.setId(saved.getId());
           }
         }
+        session.setContext(session.applyTo(x));
```

Comment:
This block should only be hit if the session ID is part of the GET url. 
DIG calls should be posting and using an auth header with BEARER Token. 
What scenario is this occurring for?

Reply:
I was hitting "Send Request" button from the GUI https://github.com/kgrgreer/foam3/blob/development/src/foam/nanos/dig/DIG.js#L496.

Reply:
Indeed, look at that. 
We should fix that so it's a proper POST, but this is needed regardless.

Reply:
So now, what is happening at line 266? Is this skipped because a subject is found at 254?

Reply:
The user at 254 is null then auth.login fails because there is no password.